set.mm

```diff
@@ -42472,6 +42448,71 @@ Equality axioms (and Extensionality) yield
   elelpwi $p |- ( ( A e. B /\ B e. ~P C ) -> A e. C ) $=
     ( cpw wcel elpwi sseld impcom ) BCDEZABEACEIBCABCFGH $.
 
+  ${
+    $d A x $.  $d B x $.
+    $( The powerclass perserves inclusion.  See ~ sspwb for the biconditional
+       version.  (Contributed by NM, 13-Oct-1996.)  Extract forward implication
+       of ~ sspwb since it requires fewer axioms.  (Revised by BJ,
+       13-Apr-2024.) $)
+    sspw $p |- ( A C_ B -> ~P A C_ ~P B ) $=
+      ( vx wss cpw cv wcel sstr2 com12 velpw 3imtr4g ssrdv ) ABDZCAEZBEZMCFZADZ
+      PBDZPNGPOGQMRPABHICAJCBJKL $.
+  $}
+
+  ${
+    sspwi.1 $e |- A C_ B $.
+    $( The powerclass perserves inclusion (inference form).  (Contributed by
+       BJ, 13-Apr-2024.) $)
+    sspwi $p |- ~P A C_ ~P B $=
+      ( wss cpw sspw ax-mp ) ABDAEBEDCABFG $.
+  $}
+
+  ${
+    sspwd.1 $e |- ( ph -> A C_ B ) $.
+    $( The powerclass perserves inclusion (deduction form).  (Contributed by
```

Comment:
```suggestion
    $( The powerclass preserves inclusion (deduction form).  (Contributed by
```

Reply:
Fixed in https://github.com/metamath/set.mm/pull/3918/commits/1d884fa9eef7707551df5107b59fa74d2bc0c268.